opendevin/runtime/od_runtime_client/od_runtime_client.py

```diff
@@ -0,0 +1,210 @@
+import asyncio
+import os
+import websockets
+import pexpect
+import json
+import shutil
+from typing import Any
+from websockets.exceptions import ConnectionClosed
+from opendevin.events.serialization import event_to_dict, event_from_dict
+from opendevin.events.observation import Observation
+from opendevin.runtime.plugins import PluginRequirement
+from opendevin.events.action import (
+    Action,
+    CmdRunAction,
+    IPythonRunCellAction,
+)
+from opendevin.events.observation import (
+    CmdOutputObservation,
+    ErrorObservation,
+    Observation,
+    IPythonRunCellObservation
+)
+from opendevin.runtime.plugins import (
+    AgentSkillsRequirement,
+    JupyterRequirement,
+    PluginRequirement,
+)
```

Comment:
These actually import types from the `opendevin` main repo. Should we make `opendevin` an pip installable package (e.g., for the types), and later install it inside the runtime sandbox?

Reply:
My current manually execution methods is mount opendevin project path into the docker, then directly run the client code.
A temporary solution is package opendevin code into the sandbox image. But make opendevin as a pip installable package is better.